### ISSUE TITLE
A little improvement for Batch Convert 

### ISSUE THREAD

**Issue by LeonCheung** [SubtitleEdit/subtitleedit#8226]
Please help add one format type that is without any extension, which is very helpful to handle files named like this:

![image](https://github.com/SubtitleEdit/subtitleedit/assets/494788/3df293f8-2005-4af4-83b7-b1c7ea740b71)

As shown above, if I use the setting like this, then I'll get file names like **xx.srt.srt** rather than **xx.srt**. which is more expected. 

A possible solution could be adding a checkbox called "**With extension**" to decide whether to add the extension or not. (and we could have it toggled by default) Thank you~

**Comment by Purfview** [comment]
> As shown above, if I use the setting like this, then I'll get file names like xx.srt.srt rather than xx.srt. which is more expected.

`xx.srt` would be incorrect an unexpected, `xx.srt.srt` is correct and expected.
Your source files are named as `xx.srt.txt`, "srt" there in the middle is not the file extension, "xx.srt" is the filename there.

> A possible solution could be adding a checkbox called "With extension" to decide whether to add the extension or not

That is nonsense.

**Comment by LeonCheung** [comment]
@Purfview Please understand that it commonly happens when you are provided with such files by clients or your team members, that's something you can hardly control. The only thing you can do is to try fixing it on your side. It's the real case I often experienced, not something I imagined.

I guess they just translated the SRT files using Notepad and then saved as TXT. But that's something I cannot control because I'm not the boss here.

**Comment by LeonCheung** [comment]
Let me explain it a bit more, maybe there could be another workaround. Here is the story:

This is a typical SRT (original ready for translation in bilingual format):
![image](https://github.com/SubtitleEdit/subtitleedit/assets/494788/54e98ec5-3842-4281-a50c-5691fc9dba77)

Unfortunately, for some reason, translation is directly written at empty lines like this:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/494788/c6f0065a-cb9c-4c46-a5de-794d2c123edd)

Here comes the problem. Although SE can successfully parse non-standard files like this, but every time a message pops up, and have to be manually closed:

![image](https://github.com/SubtitleEdit/subtitleedit/assets/494788/e98df7cc-27a4-4dcb-b6de-e652c93c1c95)

Guess what, there are hundreds of files in total! Annoying huh? So maybe a workaround is by allowing a user to turn on/off the message? I use Batch Convert as a workaround because it could fix them beforehand so that I don't have to see the message window again and again.

**Comment by Purfview** [comment]
> Please understand that it commonly happens...

I understand. And please understand that your file managing problems are out of scope for SE or any other non-file manager tool.
You can manage files with something like Total Commander, it has Multi-Rename feature.

> And that is kinda rude my friend.

I'm sorry, please don't take my honesty for rudeness.

**Comment by Purfview** [comment]
Changing the original topic of a thread like that is invalid behavior.

**Comment by LeonCheung** [comment]
@Purfview Yes I can handle it out of SE for sure. Just consider the whole thing as an end-user story. I mentioned it because SE always supports exporting custom formats without extensions. But it's fairly okay to set it as wonfix.

> Changing the original topic of a thread like that is invalid behavior.

Thanks for your honesty. I admit that I didn't quite well expose the root. But it doesn't mean I don't deserve a chance to refine the original post, right?

**Comment by Purfview** [comment]
> Thanks for your honesty. I admit that I didn't quite well expose the root. But it doesn't mean I don't deserve a chance to refine the original post, right?

Dunno what you meant as those new issues doesn't look related to your original issue.
Let me tell you a secret - open a one thread per one issue, otherwise it's hard to decipher what the actual issue is and the developers tend to ignore such threads/posts.

**Comment by LeonCheung** [comment]
Of course I know it, not a secret to me. :)  I didn't want to bother anyone by opening a new issue simply because it's still related to the root pain I open this post and I've tried to explain as much as I could.
If it still doesn't deserve a further discussion, please, let's just do nothing and keep it closed forever.

I mean why I have to face the message again and again even if I choose to trust SE that everything was successfully parsed? Precondition: for hundreds of files. 😂
